extras/claude.sh

```
@@ -0,0 +1,169 @@
+   - Known risks and mitigations
+4. Do NOT write any code yet."
+
+if [ ! -f plan.best.md ]; then
+    log "Warning: plan.best.md not produced. Falling back to plan.1.md."
+    cp plan.1.md plan.best.md 2>/dev/null || { log "Error: no plan available."; exit 1; }
+fi
+
+# Helper: run claude, display response text, return session ID via stdout
+# Usage: session=$(claude_json [extra_flags...] -- "prompt")
+claude_json() {
+    local prompt="${@: -1}"
```

Shellcheck (0.11.0)

[warning] Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.

(SC2124)